src/test/java/org/folio/rest/impl/EntitiesCrudBasicsTest.java

```diff
@@ -172,19 +173,22 @@ public void testGetRecordByIdServerError(TestEntities testEntity) {
   }
 
   @ParameterizedTest
-  @MethodSource("getTestEntitiesExceptGroupFundFYTransactionTypes")
+  @MethodSource("getTestEntitieswithGetByIdEndpoint")
   public void testGetRecordByIdNotFound(TestEntities testEntity) {
     logger.info("=== Test Get {} record by id - Not Found ===", testEntity.name());
 
     verifyGet(testEntity.getEndpointWithId(ID_DOES_NOT_EXIST), APPLICATION_JSON, NOT_FOUND.getStatusCode()).as(Errors.class);
   }
 
   @ParameterizedTest
-  @MethodSource("getTestEntitiesPostWithoutTransaction")
-  public void testPostRecord(TestEntities testEntity) {
+  @MethodSource("getTestEntitiesWithPostEndpoint")
+  public void testPostRecord(TestEntities testEntity) throws IOException {
     logger.info("=== Test create {} record ===", testEntity.name());
 
     JsonObject record = testEntity.getMockObject();
+    if(testEntity.equals(TestEntities.ORDER_TRANSACTION_SUMMARY)) {
```

Comment:
`testEntity.getMockObject()` will work correctly for ORDER_TRANSACTION_SUMMARY if you specify `ORDER_TRANSACTION_SUMMARY.jsonPathToObject = ""` instead of [`"orderTransactionSummary[0]"`](https://github.com/folio-org/mod-finance/blob/b45d0b9197aaa0af863d97b1b6453ed181eb7ea5/src/test/java/org/folio/rest/util/TestEntities.java#L45)

Reply:
Ah thanks, I was trying to get root object with / but "" works. Changed it